src/LensPlatformClient.ts

```diff
@@ -45,6 +45,25 @@ interface DecodedAccessToken {
   typ: string;
 }
 
+/**
+ * TypeGuard to determine if type is GotRequestFunction
+ */
+const isGotRequestFunction = (
+  func: any, key: keyof Got
+): func is GotRequestFunction =>
+  typeof func === "function" &&
+  ["get", "post", "put", "patch", "head", "delete"].includes(key);
+
+/**
+ * TypeGuard to determine if type is CancelableRequest | Request
+ * @remarks This is needed because for some reason the return of GotRequestFunction
+ * in got is Request | CancelableRequest
+ */
+const isCancelableRequest = (obj: any): obj is CancelableRequest => {
+  // Unlike CancelableRequest, Request doesn't have these props
```

Comment:
This could be easier to read as e.g. `// CancelableRequest has these properties unlike Request`